docs/installation-guide-ubuntu.rst

```diff
@@ -10,10 +10,10 @@ Prerequisites
 
 #. Download and install `Git (64-bit) <https://git-scm.com/download/win>`__.
 #. Download and install `Github Desktop (64-bit) <https://desktop.github.com/>`__.
-#.  Download and install `Miniconda(64-bit) for Python 2.7 <https://conda.io/miniconda.html>`__.
+#. Download and install `Miniconda(64-bit) for Python 2.7 <https://conda.io/miniconda.html>`__.
 #. Download and install `Pycharm Community edition (64-bit) <https://www.jetbrains.com/pycharm/download/#section=windows>`__.
    OR your own favorite editor.
-#. Download and install `Daysim <https://daysim.ning.com/page/download>`__.
+#. Download and install `DAYSIM version >= 4.0 <https://daysim.ning.com/page/download>`__.
```

Comment:
this step will be more complicated as there is no official ubuntu version as far as i can tell but i am busy creating the steps needed to compile it.